### PR TITLE
Add location and metatata query support

### PR DESCRIPTION
**Pull request by williamlsh**

### CODE REVIEW

File: graphql/query.go [hunk 1]
```diff
@@ -97,6 +98,24 @@ func generateCondition(valueAST ast.Value, cond db.Condition, fieldMap map[strin
...
 			} else if _, ok := fieldMap[key]; ok {
 				cond = cond.And(key, value)
+			} else if key == "location" {
+				locationObj := value.([]*ast.ObjectField)[0]
+				name := locationObj.Name.Value
+				value := locationObj.Value.GetValue()
+				if util.Contains(definition.LocationColumns, name) {
+					return cond.And("l."+name, value), nil
+				} else {
+					return db.FalseCond(), fmt.Errorf("%v not found in location", name)
```
Comment (xc) [inline graphql/query.go:108]: Outputs can be removed

File: go.mod [hunk 2]
```diff
@@ -24,5 +24,4 @@ require (
 	github.com/volatiletech/sqlboiler v3.7.1+incompatible
 	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2
 	golang.org/x/text v0.3.6
-	gopkg.in/flosch/pongo2.v2 v2.0.0-20141028005603-ca73590a271e
```
Comment (xc) [inline go.mod:27]: Removed pongo dependency?

Comment (williamlsh) [reply go.mod:27]: `github.com/flosch/pongo2 v0.0.0-20200913210552-0d938eb266f3 ` dependency already exists, `go mod tidy` detects it's redundant so removed it. But the latest version is actually `github.com/flosch/pongo2/v6`.

File: go.mod [hunk 3]
```diff
@@ -24,5 +24,5 @@ require (
...
+	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
```
Comment (xc) [inline go.mod:27]: do we need this check package?

**Pull request merged**